backend/src/mocks/mock_api_interfaces.py

```diff
@@ -67,26 +69,54 @@ def mock_pdf_attachment():
 
 def mock_chat_interface():
     return {
-        "ID": "81bddc2b-36e6-495a-a8e4-d5207a50f121",
-        "name": "First Chat",
-        "pinned": False,
-        "bookmarked": True,
+        "chatId": "81bddc2b-36e6-495a-a8e4-d5207a50f121",
+        "chatName": "First Chat - Agent 1",
+        "isOpen": False,
+        "isPinned": False,
+        "isBookmarked": True,
         "to_delete": False,
         "created_at": dt.datetime(2023, 10, 4, 12, 0, 0),
         "updated_at": dt.datetime(2023, 10, 4, 12, 0, 0),
-        "agent_id": "0e3c04dd-268a-45d8-8834-fd0e3e0c9f47",
```

Comment:
You should update the lines above to `createdAt` and `updatedAt` to be consistent.